sdcm/nemesis.py

```diff
@@ -389,6 +389,11 @@ def _kill_scylla_daemon(self):
             # Let's wait for the target Node to have their services re-started
             self.log.info('Waiting scylla services to be restarted after we killed them...')
             self.target_node.wait_db_up(timeout=14400)
+            if self.cluster.params.get('use_mgmt'):
+                # Workaround for https://github.com/scylladb/scylla-manager/issues/2813
+                # When scylla take too long time to bring api port up
+                #  scylla-manager-agent fails to start and never go up
+                self.target_node.start_service('scylla-manager-agent', timeout=600, ignore_status=True)
```

Comment:
shouldn't it be `restart` instead of `start`?

Reply:
fixed

Reply:
Nope, if it is started we need to keep it.